src/about.html

```diff
@@ -765,6 +765,210 @@ <h1 class="font-semibold text-lg sm:text-xl">
               clients we've had the pleasure of serving
             </p>
           </div>
+          <!-- cards and buttons -->
+          <div class="flex flex-col gap-10 2xl:gap-12">
+            <!-- this is for card -->
+            <div
+              class="grid grid-flow-row lg:grid-flow-col sm:gap-10 2xl:gap-12"
+            >
+              <!-- first card -->
+              <div
+                class="p-6 sm:p-10 2xl:p-12 border-button border-outline rounded-xl flex flex-col gap-7 2xl:gap-10"
+              >
+                <!-- first part -->
+                <div
+                  class="flex flex-col sm:flex-row justify-between sm:items-center gap-5"
+                >
+                  <div class="flex flex-col gap-1 sm:gap-1.5 2xl:gap-2">
+                    <p
+                      class="font-medium text-sm sm:text-base 2xl:text-lg text-color-addition-info"
+                    >
+                      Since 2019
+                    </p>
+                    <p
+                      class="text-color font-semibold text-xl sm:text-2xl 2xl:text-3xl"
+                    >
+                      ABC Corporation
+                    </p>
+                  </div>
+                  <!-- for button -->
+                  <div
+                    class="text-center px-5 py-3 2xl:py-4 2xl:px-6 text-color rounded-lg border-button bg-for-logo-section"
+                  >
+                    <a href="#">Visit Website</a>
+                  </div>
+                </div>
+                <!-- second part -->
+                <div class="grid grid-flow-col gap-5 2xl:gap-8">
+                  <div class="flex flex-col gap-1 sm:gap-1.5 2xl:gap-2">
+                    <div class="flex items-center gap-1 2xl:gap-1.5">
+                      <img
+                        src="/public/assets/aboutpage/Icon-domain.png"
```

Comment:
please start all paths with a ./ 
otherwise it will just work on local. it does not work in production  like this: ./public/assets/aboutpage/Icon-domain.png